docs/kf.dev/content/en/docs/developers/reference-apps/link-shortener/_index.md

```diff
@@ -0,0 +1,146 @@
+---
+title: "Link Shortener App "
+linkTitle: "Link Shortener App"
+weight: 20
+description: >
+  Learn how to deploy the Link Shortener reference application originally built
+  for Cloud Foundry and the Google Cloud service broker.
+---
+
+This example Spring application is an enterprise-ready link-shortener.
+The URL shortener has the following behaviour:
+
+ * URLs internal to your own domain get automatically redirected.
+ * URLs on the Internet are scanned for vulnerabilities:
+   * If the URL is malicious, the redirect is blocked.
+   * If the URL is benign, the user is shown a preview of the site and clicking
+through opens in a new tab to protect privacy.
+   * If the vulnerability scanner is down, a warning is shown.
+
+{{% alert title="Costs Money" color="warning" %}}
+Following these instructions will result in creation of a CloudSQL database instance
+that will incur billing charges on your GCP account.
+{{% /alert %}}
+
+## Prerequisites
+
+[google-broker]: /docs/operators/service-brokers/google-cloud/
+1. A Kf cluster and the `kf` CLI installed on your workstation
+1. **Google Service Broker**: Follow [these instructions][google-broker] to
+   install.
+1. **`git`**: Git is required to clone a repository.
+
+## Stage the app
+
+1. Clone the reference app source and go to the Link Shortener app's dir:
+
+```sh
+git clone https://github.com/GoogleCloudPlatform/service-broker-samples.git
+cd service-broker-samples/link-shortener
+```
+
+1. Open `manifest.yml` in your editor and edit the `env` section:
+
+  1. Set the value of `INTERNAL_DOMAIN_SUFFIX` to `"localhost:8080"`. This will
+     allow you to test the application locally after you deploy.
+
+  1. Add a `BP_JAVA_VERSION` key with a value of  `8.*` This will ensure Java 8 is used to build and run the app you deploy.
+
+    The resulting `manifest.yml` should resemble:
+
+   ```sh
+	---
+	applications:
+	- name: link-shortener
+	  memory: 1024M
+	  instances: 2
+	  env:
+		GOOGLE_API_KEY: "YOUR_API_KEY_HERE"
```

Comment:
Could we add instructions on getting the API key?

Reply:
Done